src/Supportive/DependencyInjection/DeptracExtension.php

```diff
@@ -31,18 +32,27 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('analyser', $configs['analyser']);
         $container->setParameter('use_relative_path_from_depfile', $configs['use_relative_path_from_depfile']);
         $container->setParameter('ignore_uncovered_internal_classes', $configs['ignore_uncovered_internal_classes']);
+        $container->setParameter('cache_file', $configs['cache_file']);
+
+        if ($container->hasParameter('cli.cache_file')) {
+            $container->setParameter('deptrac.cache_file', $container->getParameter('cli.cache_file'));
+        } elseif ($container->hasParameter('cache_file')) {
```

Comment:
Where do these 2 parameters (`cli.cache_file` and `cache_file`) come from?

Reply:
The idea behind adding these parameters is to distinguish between where the value for the cache-files comes from. 
Right now I ran into the problem, which value take precedence over each other. 
This should be the priority, right?  `--no-cache flag` -> `--cache-file flag`  -> `config`  ->  `default`.

I will revisit my solution today and maybe i can come up with a simpler approach.